notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<ul><li>This cell needs pep8 formatting</li><li>This seems overly complicated. Is there no simpler way to deploy a trained Pytorch model? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>It passes  pep8 check, but I updated it to broke long lines to multiple lines.</p><p> </p><p>If you mean the complexity is coming from the handler, I personally agree, but this is the design the official container expects. And it seems this is one of the standard ways of Pytorch model serving.</p><p> </p><p>Here is another example of Pytorch model serving with a custom container, but the design is basically is the same.</p><p>https://cloud.google.com/blog/topics/developers-practitioners/pytorch-google-cloud-how-deploy-pytorch-models-vertex-ai</p><p>	</p>